website/src/app/pricing/_page.tsx

```diff
@@ -371,8 +371,8 @@ export default function _Page() {
             <p>When will I be billed?</p>
           </blockquote>
           <p className="mb-8">
-            When you start service, or at the beginning of each billing cycle.
-            Enterprise plans are billed quarterly or annually.
+            The Team plan is billed when you start service, and then monthly
```

Comment:
I like the billing cycle being defined.  Just wondering if the first month is prorated or if the customer will pay for the whole month no matter when they start on the team plan?

Reply:
Gotcha, I see. Will open another Pr.